source/languages/en/riak/ops/building/installing/erlang.md

```diff
@@ -121,7 +121,8 @@ installed the necessary dependencies.
 Use this command to install the required dependency packages:
 
 ```bash
-sudo apt-get install build-essential libncurses5-dev openssl libssl-dev fop xsltproc unixodbc-dev
+sudo apt-get update
+sudo apt-get -y install build-essential autoconf libpam0g-dev maven openjdk-7-jdk
```

Comment:
Sorry, I think I led you astray a bit. I was guessing that the old way of installing was outdated, but wasn't 100% for sure that the way in the wiki I linked is the BEST way. 

So, for instance, I think maybe mixing the two is best? Perhaps maintaining the openssl and unixodbc dependencies. 

It's worth double-checking with someone in Engineering to get a thumbs-up on the recommended way (to make sure we capture all the dependencies, etc.).